worlds/hades/__init__.py

```diff
@@ -0,0 +1,255 @@
+from re import I
+import string
+import typing
+import settings
+import random
+
+from BaseClasses import Entrance, Item, ItemClassification, Location, MultiWorld, Region, Tutorial
+from .Items import event_item_pairs_weapon_mode, item_table, item_table_pacts, HadesItem, event_item_pairs, \
+      create_pact_pool_amount, create_filler_pool_options, item_table_keepsake, item_table_weapons, \
+        item_table_store, item_table_hidden_aspects, create_trap_pool, item_name_groups
+from .Locations import setup_location_table_with_settings, give_all_locations_table, HadesLocation, location_table_fates_events, location_name_groups
+from .Options import hades_options, InitialWeapon, hades_option_presets
+from .Regions import create_regions
+from .Rules import set_rules
+from worlds.AutoWorld import WebWorld, World
+from worlds.LauncherComponents import Component, components, Type, launch_subprocess
+
+
+def launch_client():
+    from .Client import launch
+    launch_subprocess(launch, "HadesClient")
+
+
+components.append(Component("Hades Client", "HadesClient",
+                  func=launch_client, component_type=Type.CLIENT))
+
+
+class HadesSettings(settings.Group):
+    class StyxScribePath(settings.UserFilePath):
+        """Path to the StyxScribe install"""
+
+    styx_scribe_path: StyxScribePath = StyxScribePath(
+        "C:/Program Files/Steam/steamapps/common/Hades/StyxScribe.py")
+
+
+class HadesWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up Hades for Archipelago. "
+        "This guide covers single-player, multiworld, and related software.",
+        "English",
+        "Hades.md",
+        "Hades/en",
+        ["Naix"]
+    )]
+    options_presets = hades_option_presets
+
+
+class HadesWorld(World):
+    """
+    Hades is a rogue-like dungeon crawler in which you defy the god of the dead as you hack and slash 
+    your way out of the Underworld of Greek myth.
+    """
+
+    option_definitions = hades_options
+    game = "Hades"
+    topology_present = False
+    data_version = 1
+    settings: typing.ClassVar[HadesSettings]
+    web = HadesWeb()
+    required_client_version = (0, 5, 0)
+
+    polycosmos_version = "0.12"
+
+    item_name_to_id = {name: data.code for name, data in item_table.items()}
+    location_table = give_all_locations_table()
+    location_name_to_id = location_table
+
+    item_name_groups = item_name_groups
+    location_name_groups = location_name_groups
+
+    def create_items(self):
+        local_location_table = setup_location_table_with_settings(self.options).copy()
+        
+        pool = []
+
+        # Fill pact items
+        item_pool_pacts = create_pact_pool_amount(self.options)
+
+        # Fill pact items
+        if (self.options.heat_system.value == 1):
```

Comment:
```suggestion
        if self.options.heat_system == "reverseheat":
```
Would recommend against using "magic numbers" for option comparisons. You can compare directly to the option name as a string. You also don't need to include `.value` for comparisons like this.

Reply:
Huh interesting, I didn't know straight comparison works for those options. Changed to this everywhere as far as I can tell. Much more redeable that way.